.github/workflows/release_wheel.yml

```diff
@@ -26,14 +26,7 @@ jobs:
       fail-fast: false
       matrix:
         cuda: ["11.8", "12.1", "12.4"]
-        torch: ["2.3", "2.4", "2.5"]
-        exclude: # for cuda 12.4, we only support torch 2.4+
-          - cuda: "12.4"
-            torch: "2.2"
-          - cuda: "12.4"
-            torch: "2.3"
-          - cuda: "12.4"
```

Comment:
keep this exclude `12.4 2.5`

Reply:
why is it the case? I saw the comment

`# for cuda 12.4, we only support torch 2.4+`

so i think cuda 12.4 + pytorch 2.5 should work.

Reply:
Since it has been taken care of. https://github.com/flashinfer-ai/flashinfer/blob/6805c643a933088d043fed86a0ec15ef76863343/.github/workflows/release_wheel_sglang.yml#L24-L26

Reply:
We use `release_wheel_sglang.yml` for faster release and verification. If everything is okay, then we trigger `release_wheel.yml`.

Reply:
sounds good, fixed now.

Reply:
BTW we should keep the `exclude:` keyword

Reply:
oops, sorry! fixed now.